git/gittest/repo.go

```diff
@@ -38,6 +38,10 @@ func Repo(t *testing.T) (git.Repo, func()) {
 		cleanup()
 		t.Fatal(err)
 	}
+	if err = execCommand("git", "-C", filesDir, "tag", "flux-test"); err != nil {
```

Comment:
Yes, the issue with this is that I think a few of the tests require a tag to be set. Although I haven't tracked them down yet. When I take this out something stops working.